Makefile

```diff
@@ -22,18 +22,24 @@ IMAGE_NAME := gcr.io/freshtracks-io/bomb-squad:$(SHORT_SHA)
 vendor/vendor.json:
 	govendor init
 
-vendor: vendor/vendor.json
+vendor: vendor/vendor.json vendor/.uptodate
```

Comment:
why is the vendor/vendor.json not enough?

Reply:
Mostly because I wanted it to run the `init` as needed, but also run every time I run `make vendor`. With just `vendor/vendor.json` it won't run again without a manual `touch vendor/vendor.json`.  There's gotta be a smarter way, but I was lazy. :-D